### PR TITLE
removed workspace viewer button

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

**Comment by 0x00b1** [comment]
> Additionally, we should comment out/remove references to the workspace viewer in the documentation to prevent confusion.

done

**Comment by 0x00b1** [comment]
> Maybe it would be better to comment this code out, so it will be easy to add back in #2909.

What if I just update the issue with the commit for the changeset?

File: cellprofiler/gui/help.py [hunk 1]
```diff
@@ -1493,186 +1493,6 @@
...
-object's pixels. Each object is assigned a color using the default color map
-initially. You can choose the color map by
-selecting <i>Color</i> from the objects' menu and choosing one of the available
-color maps. You can change the transparency of the overlay by choosing
-<i>Alpha</i> from the objects' menu.</li>
-</ul></p>
-
-""" % globals()
-
-WV_FIGURE_HELP = tuple(list(FIGURE_HELP) + [(
```
Comment (mcquin) [inline cellprofiler/gui/help.py]: Removing `WV_FIGURE_HELP` will break this: https://github.com/CellProfiler/CellProfiler/blob/e9170b747a26a1d546171cbffb6d89604b935e02/cellprofiler/gui/viewworkspace.py#L229

**Comment by mcquin** [comment]
> What if I update the issue with the commit for the changes?

That's fine, too.

File: cellprofiler/gui/help.py (hunk 1 above)
Comment (0x00b1) [reply cellprofiler/gui/help.py]: Fixed

**Comment by mcquin** [comment]
@0x00b1 This line should also be removed: https://github.com/CellProfiler/CellProfiler/blob/e9170b747a26a1d546171cbffb6d89604b935e02/cellprofiler/gui/cpframe.py#L635

**Pull request merged**